src/include/cart/types.h

```diff
@@ -60,6 +64,13 @@ typedef struct crt_init_options {
 
 	/** Used with cio_use_credits to set credit limit */
 	int		cio_ep_credits;
+
+	/**
+	 * Hints to mercury/ofi to use for expected/unexpected
+	 * maximum sizes of messages.
+	 */
+	int		cio_max_expected_size;
+	int		cio_max_unexpected_size;
```

Comment:
should be unsigned?

Reply:
fixed; was initially using -1 as 'not set', but then changed to 0 and didnt update type